packages/ifpack2/src/Ifpack2_LocalSparseTriangularSolver_def.hpp

```diff
@@ -442,6 +443,8 @@ initialize ()
       rowStart += numEnt;
       newptr(row+1) = rowStart;
     }
+
+    // FENCE REVIEW: Not Tested
```

Comment:
I don't think this fence is needed.  I suspect it was a defensive measure after the explicit use of UVM above, but I don't think it is a required one.

Reply:
I can open a PR to remove this. But I wanted to confirm if we want to do this on untested code - I have been reluctant to make any changes to untested blocks.